notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<p>For link backs, use this style (what we parse) and drop bullet list.</p><p> </p><p>Learn more about [title] (...) and [title] (...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>